sentry_kube/cli/apply.py

```diff
@@ -177,13 +177,16 @@ def _diff_kubectl(
     definitions,
     server_side=None,
     important_diffs_only: bool = False,
-) -> bool:
+    return_output: bool = False,
+):
```

Comment:
We lost our return type?
```suggestion
) -> bool:
```